scripts/weather_data_cleaning.py

```diff
@@ -0,0 +1,290 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import numpy as np
+import os
+import pandas as pd
+import requests
+
+
+def read_in_csv(url, folder_name, file_name):
+    response = requests.get(url)
+    with open(os.path.join(folder_name, file_name), 'wb') as f:
+        f.write(response.content)
+    df = pd.read_csv(folder_name + '/' + file_name)
+    return df
+
+
+def calculate_vpd(temp_avg, rh_avg):
+    es = (6.11 * np.exp((2500000/461) * (1/273 - 1/(273 + temp_avg))))
+    vpd = (((100 - rh_avg)/1000) * es)
+    return vpd
+
+
+def save_to_csv(list_of_dfs, list_of_output_filenames):
+    for i,j in zip(list_of_dfs, list_of_output_filenames):
+        i.to_csv(j, index=False)
+
+
+s4_url = 'https://de.cyverse.org/dl/d/7D6C8FD6-EF77-437C-89E6-412EA8C3EEC6/mac_weather_station_raw_daily_2017.csv'
+s6_url = 'https://de.cyverse.org/dl/d/233C21D5-1306-4028-9CF9-FF4AF0EAC405/mac_weather_station_raw_daily_2018.csv'
+ksu_hourly_url = 'https://de.cyverse.org/dl/d/D80C07D7-5F68-4C86-B15A-9BAAF472D3A4/ksu_hourly_weather.csv'
+ksu_daily_url = 'https://de.cyverse.org/dl/d/64805E3B-0460-4AA1-8D8A-2D7246E05B35/ashland_bottoms_daily_weather_2016.csv' 
+clemson_temps_url = 'https://de.cyverse.org/dl/d/19836AB5-9223-4CBA-B56E-46272CACF5A3/clemson_temps_daily.csv'
+clemson_rh_url = 'https://de.cyverse.org/dl/d/353F5386-4D88-45A9-A4F6-6FC66C64C7E8/clemson_rh_daily.csv'
+
+s4_filename = "mac_weather_station_raw_daily_2017.csv"
+s6_filename = "mac_weather_station_raw_daily_2018.csv"
+ksu_hourly_filename = "ksu_hourly_weather.csv"
+ksu_daily_filename = "ashland_bottoms_daily_weather_2016.csv" 
+clemson_temps_filename = "clemson_temps_daily.csv"
+clemson_rh_filename = "clemson_rh_daily.csv"
+
+folder_name = 'data'
```

Comment:
should create this folder if it doesn't exist

```suggestion
folder_name = 'data'
if not os.path.exists(folder_name):
    os.makedirs(folder_name)
```